src/theory/quantifiers/inst_strategy_cbqi.cpp

```diff
@@ -776,11 +779,15 @@ CegInstantiator * InstStrategyCegqi::getInstantiator( Node q ) {
 }
 
 void InstStrategyCegqi::registerQuantifier( Node q ) {
-  if( options::cbqiPreRegInst() ){
-    if( doCbqi( q ) ){
-      //just get the instantiator
+  if( doCbqi( q ) ){
+    // get the instantiator  
```

Comment:
nitpick: The comments just repeat the function names. These are not really explanatory and can be dropped.